test/acceptance/behave/components/ngsiv2/types/retrieve_entity_types.feature

```diff
@@ -0,0 +1,797 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+
+Feature: get entity types using NGSI v2 API. "GET" - /v2/types
+  As a context broker user
+  I would like to get entity types using NGSI v2 API
+  So that I can manage and use them in my scripts
+
+  Actions Before the Feature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file
+  Setup: start ContextBroker
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  Actions After each Scenario:
+  Setup: delete database in mongo
+
+  Actions After the Feature:
+  Setup: stop ContextBroker
+
+  @happy_path @BUG_1636 @skip
+  Scenario:  get entity types using NGSI v2 API
+    Given  a definition of headers
+      | parameter          | value                |
+      | Fiware-Service     | test_type_happy_path |
+      | Fiware-ServicePath | /test                |
+      | Content-Type       | application/json     |
+    And initialize the accumulator context of entities
+    And create "5" entities with "2" attributes
```

Comment:
if `entities_number` is greater than 1 then `entities_id` is a prefix. Ex: room1_0, ..., room1_4. 
See https://github.com/telefonicaid/fiware-orion/tree/develop/test/acceptance/behave#hints
This behaviour is similar in `attributes_name`.